src/CMakeLists.txt

```diff
@@ -6,20 +6,21 @@ target_sources(timg PRIVATE
 
   buffered-write-sequencer.h buffered-write-sequencer.cc
   display-options.h
```

Comment:
I'd keep the original column indentation from before here: Aligh everything that they are nicely aligned and make an exception for unicode-block-canvas

Reply:
(solved)